src/components/views/styl/Switch.styl

```diff
@@ -0,0 +1,56 @@
+@import 'includes/variables';
+
+.switch-wrapper {
+    display: inline-block;
+}
+
+.switch-content {
+    align-self: center;
+    display: flex;
+    font-size: 12px;
+    justify-content: space-between;
+    line-height: 20px;
+    cursor: pointer;
+
+    .handle {
+        background-image: linear-gradient(-180deg, #FAFAFA 0%, #EFF1F4 100%);
+        border-radius: 30px;
+        border: 1px solid rgba(0,0,0,0.1);
+        box-shadow: 0 3px 8px 0 rgba(0,0,0,0.10), inset 0 1px 0 1px #fff;
+        height: 20px;
+        left: 0;
+        position: absolute;
+        transition: left 0.2s ease;
+        width: 20px;
+    }
```

Comment:
Not a big deal, but the handle looks a little weird here. Like it's not vertically-positioned correctly.

![](https://user-images.githubusercontent.com/933396/33450080-af3099d0-d5d8-11e7-88c0-8a3f5d992ec1.png)

Reply:
It's an analog stick-switch! OK fine it wasn't well aligned.

Reply:
Heh. #skeuomorphism